counting-messages-in-a-topic/README.adoc

```diff
@@ -1,10 +1,10 @@
 = Counting messages in a topic
 
-See http://www.confluent.io/stream-processing-cookbook/ksql-recipes/counting-messages-in-a-topic
+See http://www.confluent.io/stream-processing-cookbook/ksql-recipes/counting-messages-topic
```

Comment:
Not sure why this needs to change? `in a` makes more sense, and is also the path used in the folder, if we change this we need to change the github structure too (but I don't think we should)

Reply:
I've suggested the change as we've left out "stop words" in the other cookbook URLs, but I am fine with keeping `in a` if it makes more sense.